canonical-web-and-design/ubuntu.com
# Build account page

## Done

- Create an empty /account page

## QA

- go to https://ubuntu-com-10004.demos.haus
- sign in
- check that https://ubuntu-com-10004.demos.haus/account exists and displays your email


## Screenshots

![image](https://user-images.githubusercontent.com/11927929/124946767-942c6200-e00f-11eb-9469-a5df8a833d54.png)
